api/v3/Case.php

```diff
@@ -218,6 +218,12 @@ function _civicrm_api3_case_create_spec(&$params) {
     'title' => 'Activity Medium',
     'type' => CRM_Utils_Type::T_INT,
   ];
+  $params['standard_timeline'] = [
+    'api.default' => 1,
+    'title' => 'Standard Timeline?',
+    'description' => 'Add activities from the Standard Timeline (default = yes). If No, the case will have no activities.',
```

Comment:
```suggestion
    'description' => 'Add activities from the Standard Timeline (default = "Yes"). If "No", the case will have no activities, and the caller will be responsible for adding any basic activities (eg "Open Case").',
```

Reply:
I see where you're going because my first draft included something along those lines but my two cents is merge as-is, look at making fuller documentation elsewhere, then add a link to it in here.

Reply:
My point being you can delete all the activities from the case type config in the UI without any warnings either.

Reply:
>... look at making fuller documentation elsewhere, then add a link to it in here.

Ahhh. I was thinking that there was no suitable+extant page (*none that I can think of/find*), but you're right that one could make a new page. Much like [Dev Guide: Financial](https://docs.civicrm.org/dev/en/latest/financial/overview/) documents a developer perspective on `Order` API and `Payment` API, there could be similar tutorial content for working with CiviCase-related APIs.  The scale+utility of such a page seems comparable. 

It is a bigger task than tweaking the `description`...

I suppose any one of these would suitably help to improve the comm's:

* Make the `description` more verbose
* Write a CiviCase dev chapter
* File an issue for `civicrm-dev-docs` mentioning that we need a CiviCase dev chapter

Reply:
I'm not strongly opposed to the suggested additions, and also am willing to help write some dev docs.